simulator/infra/instrcache/instr_cache_memory.h

```diff
@@ -44,7 +44,7 @@ class InstrMemory : private FuncMemory
         void store( const Instr& instr)
         {
             instr_cache.erase( instr.get_mem_addr());
-            write(instr.get_v_src2(), instr.get_mem_addr(), instr.get_mem_size());
+            write(static_cast<uint64>(instr.get_v_src2()), instr.get_mem_addr(), instr.get_mem_size());
```

Comment:
There is a potential bug for RISCV128 — it should store 128 bits.
Please add a comment here.

Reply:
I will fix it

Reply:
Let's not fix it now, as it may require some extra exercises with 128 bit in FuncMemory class.
I'll handle it later.